desktop/apps/auction2/components/Layout.js

```diff
@@ -39,7 +39,7 @@ function Layout (props) {
           <div className='auction2-main-page'>
             { showFollowedArtistsRail &&
               <WorksByFollowedArtists /> }
-            <CommercialFilter />
+            <ArtworkBrowser />
```

Comment:
This is going to soon be shared with `/auctions` so restructured slightly to make more portable